### ISSUE TITLE
Highlight what columns have queries applied

### ISSUE THREAD

**Issue by brynwhyman** [silverstripe/silverstripe-ckan-registry#170]
_This probably needs to be thought about further..._

It's not clear when you look at the GridField data table in the CMS what rows (reflecting a data column) have been edited to have extra queries applied to them. This would require clicking each to 'edit' if you wanted to check.

Suggest there is an icon displayed on the row, if there have been queries applied, to indicate at a glance which have been edited.

**Comment by ScopeyNZ** [comment]
Yeah it'll be nice to have a simple design guideline for this enhancement. /cc @sachajudd
